womtool/src/test/resources/validate/biscayne/valid/object_struct/object_struct.wdl

```diff
@@ -1,4 +1,4 @@
-version development
+version development-1.1
```

Comment:
Do we need equivalent tests for the new `development` version?

Reply:
Has the `development` version gone away? If so, some people's WDLs may break.

... on the other hand, some people's WDLs may break anyway, and it's literally called "development" so maybe we don't care?

Reply:
Note: this is a different version declaration than the one listed in `wdl/transforms/biscayne/src/main/resources/CHANGELOG.txt`. Presumably this one is correct and the changelog needs updating?

Reply:
Chris, I updated the description with info about the version declaration.